MetaMask/metamask-extension
# UX: Avoid Popover Footer when No Connected Sites

## Explanation

I noticed an unwanted grey line when there a no connected sites within the Connected Sites popover. This PR removes it.

## Screenshots/Screencaps

### Before

<img width="357" alt="SCR-20230714-sfeb" src="https://github.com/MetaMask/metamask-extension/assets/46655/598b7cab-7b38-445b-9951-38e203a21715">


### After

<img width="372" alt="SCR-20230714-sipk" src="https://github.com/MetaMask/metamask-extension/assets/46655/f20805fb-eb01-4336-97e7-a593fd7490af">


## Manual Testing Steps

1.  Go to a site without a connection
2. From the popover, click the Connected Sites Icon
3. See no footer line

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
